pymc3/linear_models/linear.py

```diff
@@ -87,7 +87,7 @@ def from_formula(cls, formula, data, priors=None, vars=None, name='', model=None
                    priors=priors, vars=vars, name=name, model=model)
 
 
-class Glm(LinearComponent):
+class GLM(LinearComponent):
```

Comment:
One Python mantra I read said to keep abbreviations upper-case as it leads to weird names, e.g. `HTTPProxy` rather than `HttpProxy`.